index.html

```diff
@@ -19,12 +19,18 @@
       <a class="name" href="https://github.com/Cluab">Ibrahim Al-Yousefi</a>
     </div>
     <nav>
-      <img class="toolbar"src="img/Union.svg" alt="side-bar">
+      <!-- <img class="toolbar"src="img/Union.svg" alt="bar"> -->
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+     </div>
       <ul class="ul-nav">
-     <li><a class="li-nav" href="index.html">Portoflio</a></li> 
-     <li><a class="li-nav" href="">About</a></li>
-     <li><a class="li-nav" href="">Contact</a></li>
-     <li><img class="nav-mail" src="img/mail.png" alt="contact mail"></li>
+     <li class="nav-li"><a class="nav-link li-nav" href="index.html">Portfolio</a></li> 
+     <li class="nav-li"><a class="li-nav nav-link" href="#about">About</a></li>
+
+     <li class="nav-li"><a class="li-nav nav-link" href="#contact">Contact</a></li>
```

Comment:
- [ ] I think it will be better if you use the correct font size and the right font weight for your mobile menu to match the Figma design 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/194938401-391d0e65-bda9-4940-b7d2-053f459b909b.png)  | ![image](https://user-images.githubusercontent.com/63473584/194938466-1f25b029-80a1-40fd-b4a2-c96170bef334.png)  |